apps/ledger-live-desktop/tests/specs/speculos/manage.ledgersync.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { test } from "../../fixtures/common";
+import { AppInfos } from "tests/enum/AppInfos";
+
+const app: AppInfos = AppInfos.LS;
+
+test.describe(`[${app.name}] Sync Accounts`, () => {
+  test.use({
+    userdata: "ledgerSync",
+    speculosApp: app,
+  });
+
+  test(
+    "Synchronize one instance then delete the backup",
+    {
+      annotation: {
+        type: "TMS",
+        description: "B2CQA-2292,B2CQA-2293,B2CQA-2296",
```

Comment:
Maybe we should add a " " between each ticket
```suggestion
        description: "B2CQA-2292, B2CQA-2293, B2CQA-2296",
```